tests/online_ddl/conf/dm-task.yaml

```diff
@@ -68,6 +70,14 @@ column-mappings:
     target-column: "id"
     arguments: ["2", "", "t"]
 
+filters:
+  filter-rule-index:
+    schema-pattern: "*"
+    table-pattern: "*"
+    events: ["create index","drop index"]
+    sql-pattern: ["ALTER\\s.*TABLE[\\s\\S]*ADD\\s+KEY", "ALTER\\s.*TABLE[\\s\\S]*DROP\\s+KEY"]
```

Comment:
How about filter index statement for special name, so we will have positive and negative test.

Reply:
what's the "negative", do you mean filter a SQL that parser can't parse, and test `handle-error skip` it?